mantidproject/mantid
# Performance optimisations for detector Banks

This PR is related to the overarching feature branch #21230 for refactoring the instrument view to use `ComponentInfo/DetectorInfo`.  These changes add optimisations for Banks of tubes, Rectangular banks and structured banks.

**To test:**
* Thorough Code Review
* Launch Instrument View and ensure that Rectangular banks (RectangularDetector e.g SANS2D, LOQ), Banks of Tubes (e.g WISH, LARMOR) and Structured banks (e.g LOKI) are loaded and displayed correctly. 

Fixes #21582 

**Release Notes** 
Release notes updated for 3.13

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
